src/test/java/com/snack/news/controller/SlackBotControllerTest.java

```diff
@@ -0,0 +1,85 @@
+package com.snack.news.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.snack.news.domain.slack.SlackChannel;
+import com.snack.news.exception.NewsNotFoundException;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'com.snack.news.exception.NewsNotFoundException'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=5471424)